SonarQube.TeamBuild.PreProcessor/SonarLintAnalyzerProvider.cs

```diff
@@ -29,6 +36,8 @@ public static class SonarLintAnalyzerProvider
         public const string CSharpPluginKey = "csharp";
         public const string CSharpRepositoryKey = "csharp";
 
+        public const string EmbeddedSonarLintZipFileName = "SonarLint.zip";
```

Comment:
We don't yet include the SonarLint.zip file in the C# plugin. However, if you manually zip SonarLint.dll and add it to the "static" folder of the jar file, the contents will be downloaded and unpacked.
The code will just write out a debug message if the zip can't be found (which shouldn't happen in production as the new version pre-processor will only ship in a version of the C# plugin that has the SonarLint.dll embedded in it).